tests/pytorch_tests/function_tests/model_gradients_test.py

```diff
@@ -99,17 +99,46 @@ def __init__(self):
         self.conv = Conv2d(3, 3, kernel_size=1, stride=1)
         self.bn = BatchNorm2d(3)
         self.relu = ReLU()
-        # self.soft = torch.nn.Softmax(dim=1)
 
     def forward(self, inp):
         x = self.conv(inp)
         x = self.bn(x)
         x = self.relu(x)
-        # x = self.soft(x)
         x = torch.argmax(x)
         return x
 
 
+class node_with_multiple_outputs_model(torch.nn.Module):
+    def __init__(self):
+        super(node_with_multiple_outputs_model, self).__init__()
+        self.conv1 = Conv2d(3, 3, kernel_size=(1, 1), stride=(1, 1))
+        self.bn = BatchNorm2d(3)
+        self.relu = ReLU()
+
+    def forward(self, inp):
+        x = self.conv1(inp)
+        x = self.bn(x)
+        x = self.relu(x)
+        y, z, w = torch.split(x, split_size_or_sections=1, dim=1)
+        return y + z + w
```

Comment:
didn't you mean return y, z, x?

Reply:
It doesn't really matters for the test, but no I meant to use "add" anyway